codelabs/Utopia Wholesale Codelab/2-holder.md

```diff
@@ -225,12 +225,22 @@ if (DigitalCredentials.Default.available) {
 
 The Holder app also needs to add the Verifier (Reader) certificate to its trust list. This ensures that the Holder can recognize and trust the Verifier during credential sharing. The Verifier's certificate can be downloaded from the Multipaz Verifier [website](https://verifier.multipaz.org/identityreaderbackend/). Below is the code snippet demonstrating how to add the Verifier's certificate to the trust list:
 ```
-   addTrustPoint(
+        addTrustPoint(
                     TrustPoint(
                         certificate = X509Cert.fromPem(
-                            getReader_Root_Cert().trimIndent().trim() # Verifier's cert
+                            getReader_Root_Cert().trimIndent().trim()
                         ),
-                        displayName = "OWF Multipaz TestApp",
+                        displayName = "OWF Multipaz Reader App",
+                        displayIcon = null,
+                        privacyPolicyUrl = "https://apps.multipaz.org"
+                    )
+                )
+                addTrustPoint(
+                    TrustPoint(
+                        certificate = X509Cert.fromPem(
+                            getReader_Root_Cert_for_untrust_device().trimIndent().trim()
+                        ),
+                        displayName = "OWF Multipaz Reader App",
                         displayIcon = null,
                         privacyPolicyUrl = "https://apps.multipaz.org"
```

Comment:
Can this point to this link instead? https://verifier.multipaz.org/identityreaderbackend/